RPIPs/RPIP-45.md

```diff
@@ -1,59 +0,0 @@
----
-rpip: 45
-title: RPL Burn
-description: Direct revenue surplus from borrowed ETH to buy and burn permissionlessly provided RPL.
-author: Valdorff (@Valdorff)
-discussions-to: TBD
-status: Draft
-type: Protocol
-category: Core
-created: 2024-03-08
-tags: tokenomics-2024, tokenomics-value-capture-option
----
-
-## Abstract
-This proposal establishes a contract that holds ETH from protocol revenue and allows for RPL to be burned in exchange for that protocol ETH. This provides direct utility to the RPL token.
-
-
-## Specification
-- A share of the revenue from borrowed ETH SHALL be directed to a smart contract (the burn contract) for this burn (see [RPIP-46](RPIP-46.md))
-- Any user MAY call a function in the burn contract to get ETH from the burn contract in exchange for RPL
-- Any RPL exchanged this way SHALL burned using the RPL token's `burn` or `burnFrom` functions
-- The swap price SHALL be based on an on-chain oracle
```

Comment:
Need some margin on here if you expect this to be called regularly.

Reply:
The premise was that the on-chain oracle, being on-chain, would need to use "time" averaging of some kind. So any time that there's a move down, this price will be a little outdated and thus provide margin.